src/components/Layouts/Stack.js

```diff
@@ -0,0 +1,39 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+
+import './Stack.css';
+
+export const ORIENTATION = {
+  vertical: 'vertical',
+  horizontal: 'horizontal'
+};
+
+class Stack extends PureComponent {
+  getFlow = () => {
+    const { orientation } = this.props;
+
+    return orientation === ORIENTATION.vertical ? 'column' : 'row' ;
+  }
+
+  render() {
+    const { children } = this.props;
+
+    const flow = this.getFlow();
```

Comment:
¿Por qué no hacer aquí directamente `orientation === ORIENTATION.vertical ? 'column' : 'row' ;`?

Reply:
¿Why not doing it here directly: `orientation === ORIENTATION.vertical ? 'column' : 'row' ;`?